spec/controllers/users_controller_spec.rb

```diff
@@ -1,6 +1,19 @@
 require "rails_helper"
 
 describe UsersController do
+  context "GET #index" do
+    context "when not logged in" do
+      it "won't access the index page" do
+        create(:user, admin: true)
```

Comment:
Why do you need to create a user here? and why do they need to be admin?

Reply:
I don't, sorry, done in sha: 08e222e